app/src/test/java/ar/scacchipa/jsonplaceholderviewer/PostRespositoryUnitTest.kt

```diff
@@ -0,0 +1,84 @@
+package ar.scacchipa.jsonplaceholderviewer
+
```

Comment:
the class name refers to one only of the repositories, but includes testing for the two of them, maybe a bettter name could be. found in case you want to put all the test in just this file.